static/js/custom-window.ts

```diff
@@ -0,0 +1,5 @@
+import ClassicEditor from "@ckeditor/ckeditor5-build-classic";
+
+export interface CustomWindow extends Window {
```

Comment:
I think perhaps we could give this interface a more descriptive name, because it's not clear right now the purpose of it. If it's just meant to be used to hold the ckEditor, then perhaps `CKWindow` could be a fine option? What do you think?

Reply:
No, it's meant to be used overall in TS. This due to the fact that TS will complain when we try to attach variables to the window object! So, custom window sounded logical to me. I don't mind renaming it if you have other suggestions! Maybe i could add a comment in that file to explain its purpose?